ConnectorTeklaStructures/ConnectorTeklaStructuresShared/PolygonMesherReferencer.cs

```diff
@@ -0,0 +1,16 @@
+using StructuralUtilities.PolygonMesher;
+
+namespace ConnectorTeklaStructuresShared
+{
+  internal class PolygonMesherReferencer
+  {
+    public PolygonMesher UnusedMethod()
```

Comment:
Hmm I saw the comment but this looks dodgy...

@JR-Morgan what do you think about this? The problem it describes is legit though

Reply:
After a quick chat with Jedd we're assuming this is to introduce a forced usage of `PolygonMesher` to prevent it's accidental removal in the future?

Waiting on confirmation of this and I'll approve :)